### ISSUE TITLE
http-auth doesn't seem to work

### ISSUE THREAD

**Issue by YasenMakioui** [dutchcoders/transfer.sh#571]
I've been trying to run the transfersh Docker image using --http-auth-user and http-auth-pass.

It's suposed to ask for credentials on upload, but it doesn't.

This didn't happen over 5/6 months ago since I ran the same command and the basic-auth was working fine.

The command I'm executing is:

```shell
docker run --publish 8080:8080 dutchcoders/transfer.sh:latest --provider local --basedir /tmp --http-auth-user test http-auth-pass test
```

**Comment by aspacca** [comment]
please. pull `dutchcoders/transfer.sh:latest` again, this was raised on https://github.com/dutchcoders/transfer.sh/issues/563 and fixed in 1.6.0

**Comment by YasenMakioui** [comment]
Woah! Nice!

I was using another version. I tried the latest version a month or two ago.

Thanks for the support!!!!
